shopfloor/services/checkout.py

```diff
@@ -352,11 +352,14 @@ def select(self, picking_id):
             return self._response_for_manual_selection(message=message)
         return self._select_picking(picking, "manual_selection")
 
-    def _select_lines(self, lines):
+    def _select_lines(self, lines, no_prefill_qty=None):
         for line in lines:
             if line.shopfloor_checkout_done:
                 continue
-            line.qty_done = line.product_uom_qty
+            if no_prefill_qty is None:
+                no_prefill_qty = line.product_uom_qty
+            line.qty_done = self._get_prefill_qty(line, no_prefill_qty)
```

Comment:
I'm not sure to understand this parameter `no_prefill_qty`, is it a boolean or a qty? Just talking about the name, should it be renamed?

Reply:
It is a quantity that represents the quantity to add to `qty_done` when the `no_prefill_qty` option is set to False
Do you think it needs to be renamed? Do you have a suggestion?

Reply:
It has been changed into prefill_qty